src/main/java/net/glowstone/entity/objects/GlowMinecart.java

```diff
@@ -27,8 +29,17 @@
 import org.bukkit.material.MaterialData;
 import org.bukkit.util.Vector;
 
+// TODO: Minecarts stop when there's an arrow in the way.
```

Comment:
Since Minecart behavior is essentially not implemented, I think this todo is too specific. Movement needs to be implemented, then collision checks with other entities (not just arrows) as well.